2021q2/git.adoc

```diff
@@ -0,0 +1,49 @@
+link:http://gameoftrees.org/[Game of Trees] URL: link:http://gameoftrees.org/[http://gameoftrees.org/] +
+link:https://github.com/johnmehr/gitup[gitup] URL: link:https://github.com/johnmehr/gitup[https://github.com/johnmehr/gitup]
+
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org> +
+Contact: Warner Losh <imp@FreeBSD.org> +
+Contact: Ed Maste <emaste@FreeBSD.org> +
+Contact: Ulrich Spörlein <uqs@FreeBSD.org> +
+Contact: link:https://lists.freebsd.org/mailman/listinfo/freebsd-git[FreeBSD-git mailing list] +
+Contact IRC #gitcvt channel on EFnet
+
+The Git Working Group has been working on ports repository migration to Git, this task started at the end of the March, beginning with a final Subversion commit on March 31st to indicate that the conversion started.
+The whole migration completed on April 6th.
+Since 2021Q2, the ports quarterly branch is created in Git repository only.
+We continued working on portsnap and other ports infrastructure to accommodate git.
+
+We continued working on implementing and updating commit hooks.
+The FreeBSD 13.0 release process was using Git
+And we are sorting and making the infrastructure to public available, as well as the documents.
+
+On June 8th, we worked with out ZFS developers to have better tracking of upstream OpenZFS development.
+The vendor/openzfs branch was renamed to vendor/openzfs/legacy.
+Two new branches were imported directly from upstream, vendor/openzfs/master and vendor/openzfs/zfs-2.1-release, and merged to main and stable/13.
+The details and the required action to correct the errors might result for the people tracking the old branche is available at link:https://lists.freebsd.org/archives/freebsd-current/2021-June/000153.html[https://lists.freebsd.org/archives/freebsd-current/2021-June/000153.html]
+
+The Git Working Group continues to track progress on two permissively-licensed git compatible tools: Gitup and Game of Trees.
+Gitup is a small, dependency-free tool to clone and update git repositories.
+It is used only to keep a local tree up-to-date, and has no support for local commits.
+
+Game of Trees is a version control client that is compatible with Git repositories.
+It provides a user interface and workflow that is distinct from that of Git.
+It is in no way intended to be a drop-in replacement for git, but can be used to develop software maintained in a Git repository.
+
+link:https://www.freshports.org/net/gitup[Gitup] and link:https://www.freshports.org/devel/got[Game of Trees] are currently available as ports and packages.
+Future work will evaluate them as candidates for the base system.
+
+The core team begin a new effort to investigate and evaluate new workflow changes in the June 2021 DevSummit.
+In the third quarter of 2021 we expect to complete the remaining migration tasks and reform the workgroup to help the workflow refresh.
```

Comment:
I'd suggest
"In the third quarter of 2021 we expect to complete the remaining migration tasks."

that part is good.

"and reform the workgroup to help the workflow refresh."

Reform isn't quite the right word.  We'll be creating a new working group to deal with the workflow. So I'd suggest:

"and create a new working group to help with workflow refresh."

I think is a better way to frame this.

Reply:
I'd also suggest adding that we've wound up our regular meetings.  The remaining migration tasks are being done by individuals (such as yourself). The new working group(s) will have people that participated in this working group as well as new people who will bring new perspectives to the process.